client/blocks/reader-subscription-list-item/index.jsx

```diff
@@ -63,6 +65,23 @@ function ReaderSubscriptionListItem( {
 		return <ReaderSubscriptionListItemPlaceholder />;
 	}
 
+	function recordEvent( name ) {
+		const props = {
+			blog_ID: siteId,
+			feed_ID: feedId,
+		};
+		if ( railcar ) {
+			recordTrackWithRailcar( name, railcar, props );
+		} else {
+			recordTrack( name, props );
+		}
+	}
+
+	const recordTitleClick = () => recordEvent( 'calypso_reader_feed_link_clicked' );
+	const recordAuthorClick = () => recordEvent( 'calypso_reader_author_link_clicked' );
+	const recordSiteUrlClick = () => recordEvent( 'calypso_reader_recommended_site_clicked' );
```

Comment:
this is on the generic ReaderSubscriptionListItem component which doesn't always correspond with a recommended sites.  e.g. this is getting fired from clicking the site for a current sub